src/Groups/matrices/iso_oscar_gap.jl

```diff
@@ -89,7 +89,7 @@ function mat_oscar_gap(x::fq_nmod_mat, riso)
       S[i] = GAP.julia_to_gap([riso(x[i,j]) for j in 1:n])
    end
 
-   return GAP.Globals.ImmutableMatrix( riso.codomain, GAP.julia_to_gap(S), true)
+   return GAP.julia_to_gap(S)
```

Comment:
I think this is an accidental change?

Reply:
If we decide to maintain `GAP.Globals.ImmutableMatrix`, yes. I'll restore it.

Reply:
Please restore it